tools/proto_format/proto_sync.py

```diff
@@ -427,8 +427,6 @@ def sync(api_root, mode, labels, shadow):
                     print(
                         'Proto formatting may overwrite or delete files in the above list with no git backup.'
                     )
-                    if input('Continue? [yN] ').strip().lower() != 'y':
```

Comment:
in most cases, that will be used to fix.

Reply:
my concern is that this was added for a reason - i guess because it can make a mess of your local repo - so my preference is defo to keep this i think in the situation of not being in CI

in terms of detecting ci - we could use an environment variable - there might be one already that we could use - but im trying to move away from using env vars for this kinda thing - so my pref would be to argparse a `--ci` flag or similary

Reply:
Can we use a timeout for input? https://pypi.org/project/inputimeout/ will be useful.

Reply:
perhaps - but i think thats what happens anyway - and it just errors - after a long time

im also v reluctant to add/keep anything that slows down an already painfully slow ci job

i think we want to avoid this routine altogether, but just in ci